lesson/day01.md

```diff
@@ -5,103 +5,109 @@ title: Day 1
 subtitle: Introduction
 ---
 
-## Requirements for Learning
+# Prerequisites
+  ---------------------
 
-To learn assembly programming for the TI-83 Plus, you will require a few
-things. This lesson will guide you through setting up all the required tools
-and creating your first program.
+In order to use this guide, you must have some experience with C or a similar programming 
+language. C will be used frequently in this guide as “pseudocode”.
 
-It is also highly recommended that you be fairly competent in any programming
-language. And when I say "any programming language", I mean, of course, C. If
-z80 is your first venture into programming, many portions of this tutorial
-will be confusing. I will use C as a kind of "pseudocode" from time to time,
-so if you know any kind of high-level language (with the definite exception of
-TI-BASIC) you should do okay.
+This guide also assumes that you have had enough experience with programming to know common
+programming terms, such as *directory*, *source code*, and *compiling*.
 
-## Software
+# Setup
+  ---------------------
 
-In order to begin creating your own programs, you'll need a few tools on your
-computer.
+First, you will need to create a directory on your computer’s main drive to store your
+programs and the tools you will need. The method for doing this varies between operating
+systems.
 
-### Assembler
+### Windows
+Create a directory on the C: drive of your computer and give it a descriptive name, such
```

Comment:
There's no compelling reason to choose the root of C: over a directory that the user owns (and doesn't require elevation to create; many readers will not have admin permissions).

Reply:
I was unaware that you could do that. Thanks for pointing it out!